neurons/validator.py

```diff
@@ -772,17 +787,20 @@ async def run_step(self):
 
                     with compute_loss_perf.sample():
                         # Run each computation in a subprocess so that the GPU is reset between each model.
+                        start_loss = time.time()
                         losses = utils.run_in_subprocess(
                             functools.partial(
                                 pt.validation.compute_losses,
                                 model_i.pt_model,
                                 batches,
                                 self.config.device,
                                 tokenizer.eos_token_id,
+                                pack_samples
                             ),
-                            ttl=360,
+                            ttl=400,
                             mode="spawn",
                         )
+                        bt.logging.debug(f'Finished computing loss in {time.time() - start_loss} seconds')
```

Comment:
We already use the sample which records this information right? Is it necessary to add the extra log here?

Reply:
I am not sure where this is logged. I can however remove it, I added it while I was debugging.